src/browser.ts

```diff
@@ -130,23 +130,23 @@ export class GrpcClient {
       return cancelHandler;
     }
 
-    const languageServiceStub = service.create(serviceClientImpl, false, false);
+    const serviceStub = service.create(serviceClientImpl, false, false);
     const methods = this.getServiceMethods(service);
 
-    const newLanguageServiceStub = service.create(
+    const newServiceStub = service.create(
```

Comment:
I understand this is unrelated, but still wondering: does `newServiceStub` need to be created as a `service.create()`? I'm guessing it's basically just an `{}` object with methods redefined below. Can you try assigning it as `newServiceStub = {}` and check if it still works?

Reply:
The tests fail when I do this. When I dump the resulting newServiceStub it looks like this.

`Service{_listeners: Object{}, rpcImpl: function serviceClientImpl(method, requestData, callback) { ... }, requestDelimited: false, responseDelimited: false, echo: function echo(r,c){ ... }, expand: function expand(r,c){ ... }, coll
ect: function collect(r,c){ ... }, chat: function chat(r,c){ ... }, pagedExpand: function pagedExpand(r,c){ ... }, wait: function wait(r,c){ ... }}`

It looks like it adds a few things on top of the methods themselves, and the createStub tests fail when I change this to an empty object.

Reply:
OK then - I kind of understand the benefits of having it return the service stub and not just an `{}` object, so let's leave it as is!